Firestore/core/src/firebase/firestore/core/query.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_QUERY_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_QUERY_H_
+
+#include <utility>
+
+#include "Firestore/core/src/firebase/firestore/model/document.h"
+#include "Firestore/core/src/firebase/firestore/model/resource_path.h"
+
+namespace firebase {
+namespace firestore {
+namespace core {
+
+/** Represents the internal structure of a Firestore Query. */
+class Query {
+ public:
+  static Query AtPath(model::ResourcePath path) {
```

Comment:
Please port comments (from FSTQuery.h)

Reply:
In my defence, I ported the java comments for this method exactly. :)

... and now I get to port the objc comments to java too. :(